extensions/amp-accordion/amp-accordion.md

```diff
@@ -69,10 +71,8 @@ on mobile devices.
 
 The example below contains an `amp-accordion` with three sections. The
 `expanded` attribute on the third section expands it on page load.
-[filter formats="websites, ads"]Include the `disable-session-state` attribute
-to preserve the collapsed/expanded state.[/filter]
 
-[example preview="top-frame" playground="true"]
+[example preview="top-frame" playground="true" imports="amp-accordion:1.0"]
```

Comment:
@sebastianbenz will this syntax work? Should we include it in the documentation? https://amp.dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/?format=websites#preview-code-samples